test/cql-pytest/test_wasm.py

```diff
@@ -1107,3 +1107,97 @@ def test_counter(cql, test_keyspace, scylla_only):
             cql.execute(f"UPDATE {table} SET c = c + 1  WHERE p = 42;")
             cql.execute(f"UPDATE {table} SET c = c - 4  WHERE p = 42;")
             assert cql.execute(f"SELECT {ri_counter_name}(c) AS result FROM {table} WHERE p = 42").one().result == -1
+
+def test_docs_assemblyscript(cql, test_keyspace, table1, scylla_only):
```

Comment:
Please add a comment how this test relates to the docs, and accurate information on how you compiled (?) the weird source code below, in case a future developer needs to recompile it after a change to the doc.

Reply:
The UDF source code and compilation commands are described in the docs file.
I added a comment referring to this test in the docs, and a comment referring to the docs here (in addition to the suggestive name).